index.js

```diff
@@ -213,6 +213,16 @@ app.get('/beacon/:id/visit', async function(req, res) {
   }
 });
 
+app.get('/cv_script', async function(req, res) {
```

Comment:
Can we rename to just be `cvscript`?

Reply:
Done

Reply:
We'll have to change it in the other PRs too?

Reply:
Yes, already taken care of also